src/pages/index.css

```diff
@@ -1,3 +1,280 @@
+@font-face {
+  font-family: MontserratReg;
```

Comment:
- [ ]  Kindly deploy the project online and include the live deploy link in the readme.

Reply:
>     * [ ]   Kindly deploy the project online and include the live deploy link in the readme.

This is not part of the requirements, please approve the PR.